src/commands/miscellaneous/Feedback.ts

```diff
@@ -72,21 +81,86 @@ export default class FeedbackCommand extends BaseCommand {
 			modalInteraction.fields.getTextInputValue("feedback-input");
 
 		const embed = new EmbedBuilder()
-			.setTitle(`bois we got some feedback`)
+			.setTitle(`bois we got some feedback rahh`)
 			.setDescription(feedback)
 			.setColor(Colors.Blue)
 			.setAuthor({
 				name: interaction.user.tag,
 				iconURL: interaction.user.displayAvatarURL()
 			});
 
-		Logger.channel(interaction.guild, guildPreferences.feedbackChannelId, {
-			embeds: [embed]
-		});
+		const selectCustomId = uuidv4();
 
-		await modalInteraction.reply({
-			content: "Feedback sent!",
+		const teamSelect = new Select(
+			"team",
+			"Select a team to send the feedback to",
+			feedbackTeams.map(({ label, id }) => ({
+				label: `${label}`,
+				value: id
+			})),
+			1,
+			`${selectCustomId}_0`
+		);
+
+		const selectInteraction = await modalInteraction.reply({
+			content: "Feedback acknowledged. Select a team to send feedback to",
+			components: [
+				new ActionRowBuilder<Select>().addComponents(teamSelect),
+				new Buttons(selectCustomId) as ActionRowBuilder<ButtonBuilder>
+			],
+			fetchReply: true,
 			ephemeral: true
 		});
+
+		const response = await teamSelect.waitForResponse(
+			`${selectCustomId}_0`,
+			selectInteraction,
+			interaction,
+			true
+		);
+
+		if (!response || response === "Timed out" || !response[0]) {
+			await interaction.followUp({
+				content: "An error occurred",
+				ephemeral: false
+			})
+			return;
+		}
+
+		const team = await FeedbackChannels.findById(response[0]);
+
+		if (!team) {
+			await interaction.followUp({
+				content: "Team not found",
+				ephemeral: false
+			})
+			return;
+		}
+
+		try {
+			Logger.channel(client.guilds.cache.get(process.env.MAIN_GUILD_ID)!, team.channelId, {
```

Comment:
Don't use `!` even though we know it exists. Just fetch it above and use an if statement.